tests/fail-dep/concurrency/windows_join_detached.stderr

```diff
@@ -11,7 +11,7 @@ LL |         let rc = unsafe { c::WaitForSingleObject(self.handle.as_raw_handle(
    = note: inside `std::thread::JoinInner::<'_, ()>::join` at RUSTLIB/std/src/thread/mod.rs:LL:CC
    = note: inside `std::thread::JoinHandle::<()>::join` at RUSTLIB/std/src/thread/mod.rs:LL:CC
 note: inside `main`
-  --> $DIR/windows_join_detached.rs:LL:CC
```

Comment:
Why is this not being normalized any more?

Reply:
`https://github.com/oli-obk/ui_test/blob/0.22.0/CHANGELOG.md#removed` https://github.com/oli-obk/ui_test/pull/180 but probably can be restored.

Reply:
Yeah we probably want to add that normalization ourselves then.

Reply:
Added workaround, now it's works almost as before.